apis/datastream/v1alpha1/types.generated.go

```diff
@@ -54,10 +295,128 @@ type OracleSSLConfig struct {
 	CACertificate *string `json:"caCertificate,omitempty"`
```

Comment:
I'm not sure if we should use k8s secret here. Seems to be a bad idea to store PEM-encoded cert via GitOps.